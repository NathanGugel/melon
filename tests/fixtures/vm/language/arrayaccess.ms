let a = [1, 2, 3, 4];
io.print(a[2]);

let obj = { testProp1 = true, testProp2 = "correct" };
io.print(obj["testProp1"]);
io.print(obj["testProp" .. string.toString(a[1])]);

let str = "thisIsMyString";
io.print(str[7]);
io.print(str[0]);

a[2] = a[2] + 1;
io.print(a[2]);

obj["testProp1"] = !obj["testProp1"];
io.print(obj.testProp1);