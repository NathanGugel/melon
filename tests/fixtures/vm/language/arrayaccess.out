3
true
correct
y
t
4
false
